pkg/services/sqlstore/dashboard_version.go

```diff
@@ -58,3 +66,92 @@ func GetDashboardVersions(query *m.GetDashboardVersionsQuery) error {
 	}
 	return nil
 }
+
+func DeleteExpiredVersions(cmd *m.DeleteExpiredVersionsCommand) error {
```

Comment:
A sql integration test (like we have for the other db stuff) would be very useful here.